bootstrap/bootstrap.go

```diff
@@ -1157,11 +1171,11 @@ func (b *Bootstrap) defaultCheckoutPhase() error {
 	}
 
 	if b.Commit == "HEAD" {
-		if err := b.shell.Run("git", "checkout", "-f", "FETCH_HEAD"); err != nil {
+		if err := gitCheckout(b.shell, `-f`, `FETCH_HEAD`); err != nil {
```

Comment:
Out of interest, what's up with the new ` party? Are they better?

Reply:
@keithpitt I use `` for literals.